relops_hardware_controller/settings.py

```diff
@@ -212,7 +212,7 @@ def CACHES(self):
         api_key='${api_key}',
         product='Infrastructure & Operations',
         component='CIDuty',
-        summary='${hostname} problem tracking',
+        summary='[${DC}] ${alias} problem tracking',
```

Comment:
Do you think we can keep the FQDN in the summary or maybe include it in the bug comment?  It makes it easier to c&p when going to troubleshoot.

Reply:
:+1: I'll make another PR to add some details including the FQDN into the tracker bug comments.
The hostname will still be in the "unreachable" bug summary but not in the CIDuty tracker bug.